### PR TITLE
Correcting isntall

### PR DESCRIPTION
**Pull request by tomazcunha**
# Correcting a typo for the install command

When using the command 'npm install --help', I noticed that in the description 'aliases: i, isntall, add', install is written isntall. I imagine that this is some type of typo, so looking at the code, I made some changes that I believe can help. Thank you.

### CODE REVIEW

File: lib/config/cmd-list.js [hunk 1]
```diff
@@ -28,9 +28,8 @@ var affordances = {
   'verison': 'version',
   'ic': 'ci',
   'innit': 'init',
-  'isntall': 'install',
+  'install': 'install',
```
Comment (ljharb) [inline lib/config/cmd-list.js:31]: this is very much not a typo; it's an attempt for npm to fix a *user's* typo.

File: lib/npm.js [hunk 2]
```diff
@@ -92,7 +92,7 @@
   var commandCache = {}
   var aliasNames = Object.keys(aliases)
 
-  var littleGuys = [ 'isntall', 'verison' ]
+  var littleGuys = [ 'install', 'verison' ]
```
Comment (ljharb) [inline lib/npm.js:95]: you missed "verison" which is a misspelling of "version" - but these are also intentional.

these little guys? i wouldn't worry about these little guys.

File: lib/config/cmd-list.js (hunk 1 above)
Comment (tomazcunha) [reply lib/config/cmd-list.js:31]: Ok. Thanks for the answer.

File: lib/npm.js (hunk 2 above)
Comment (tomazcunha) [reply lib/npm.js:95]: Ok. Thanks for the answer.

**Comment by ruyadorno** [comment]
thanks @ljharb and @tomazcunha 😊

**Pull request closed without merging**